external/test-runtime/XUnit.Runtime.depproj

```diff
@@ -78,11 +78,21 @@
     <PackageToInclude Include="xunit.extensibility.core"/>
     <PackageToInclude Include="xunit.extensibility.execution"/>
     <PackageToInclude Include="xunit.runner.utility"/>
+    <PackageToInclude Include="xunit.Sdk"/>
     <PackageToInclude Include="Microsoft.xunit.netcore.extensions"/>
     <PackageToInclude Include="Newtonsoft.Json"/>
     <PackageToInclude Include="$(XUnitRunnerPackageId)" />
   </ItemGroup>
 
+  <PropertyGroup Condition="'$(TargetGroup)' == 'uap'">
+    <UAPToolsPackageVersion>1.0.5</UAPToolsPackageVersion>
```

Comment:
While you're moving this, I think it should go in dependencies.props, in the `Full package version strings that are used in other parts of the build.` section. (No condition needed.) imo `*PackageVersion` properties belong there.